client/js/controllers/post_list_controller.js

```diff
@@ -91,6 +103,36 @@ class PostListController {
         e.detail.post.save().catch((error) => window.alert(error.message));
     }
 
+    _evtMarkForDeletion(e) {
+        const postId = e.detail.post.id;
+
+        if (e.detail.delete) {
+            this._bulkEditDelete.push(postId);
+        } else {
+            // Remove item from delete list
+            this._bulkEditDelete = this._bulkEditDelete.filter(
+                (x) => x != postId
+            );
+        }
+    }
+
+    async _evtDeleteSelectedPosts(e) {
+        if (
+            confirm(
+                `Are you sure you want to delete ${this._bulkEditDelete.length} posts?`
+            )
+        ) {
+            for (let postId of this._bulkEditDelete) {
+                const post = await Post.get(postId);
```

Comment:
There is probably a better way to get a reference to the post.

Reply:
Oh, and this (currently) also doesn't show the progress nor does it catch the errors in a nice way.